components/tidb_query/src/rpn_expr/impl_json.rs

```diff
@@ -110,8 +111,8 @@ fn json_with_bytes_validator(expr: &tipb::Expr) -> Result<()> {
     Ok(())
 }
 
-/// args should be like `&Option<Json> , &[&Option<Bytes>]`.
-#[rpn_fn(raw_varg, min_args = 2, extra_validator = json_with_bytes_validator)]
+/// args should be like `(&Option<Json> , &[&Option<Bytes>])`.
```

Comment:
repeated comments? Or you should use `Args` instead.

Reply:
> repeated comments? Or you should use `Args` instead.

I move all comments to `json_with_paths_validator` now.